Content.Server/Silicons/Borgs/BorgSystem.Transponder.cs

```diff
@@ -0,0 +1,107 @@
+using Content.Shared.Emag.Components;
+using Content.Shared.Popups;
+using Content.Shared.Research;
+using Content.Shared.Silicons.Borgs.Components;
+using Content.Server.DeviceNetwork;
+using Content.Server.DeviceNetwork.Components;
+using Content.Server.DeviceNetwork.Systems;
+using Content.Server.Explosion.Components;
+
+namespace Content.Server.Silicons.Borgs;
+
+/// <inheritdoc/>
+public sealed partial class BorgSystem
+{
+    private void InitializeTransponder()
+    {
+        SubscribeLocalEvent<BorgTransponderComponent, DeviceNetworkPacketEvent>(OnPacketReceived);
+    }
+
+    public override void Update(float frameTime)
```

Comment:
Shared

Reply:
device network is server noob

Reply:
Most of it needs moving to shared and idk why it was dumped on server but this one is probably fine.